tox.ini

```diff
@@ -74,4 +74,7 @@ commands =
     {toxinidir}/scripts/run_regression.sh
 deps =
     unittest2
+#   Use a development checkout of oauth2client until a release is made
+#   which fixes https://github.com/google/oauth2client/issues/125
+    -egit+https://github.com/google/oauth2client.git#egg=oauth2client
```

Comment:
Why the `-e`? Is it a `tox` thing or an `egg` thing?

`pip` would be happy with just `git+https://github.com/google/oauth2client.git`